apps/testingaccessibility/src/components/team/buy-more-seats.tsx

```diff
@@ -1,21 +1,84 @@
 import React from 'react'
+import {z} from 'zod'
+import type {FormattedPrice} from '@skillrecordings/commerce-server/dist/@types'
+import {useQuery} from 'react-query'
+import {useDebounce} from '@skillrecordings/react'
 
-const BuyMoreSeats = () => {
-  const handleOnSubmit = () => {}
-  const calculatedPrice = 0
+const buildFormActionPath = (params: {
+  userId: string
+  quantity: number
+  productId: string
+}) => {
+  const {productId, quantity, userId} = params
+
+  const queryParamString = new URLSearchParams({
+    productId,
+    quantity: String(quantity),
+    userId,
+  }).toString()
+
+  return `/api/skill/checkout/stripe?${queryParamString}`
+}
```

Comment:
build the path's query params with `URLSearchParams` because it is easier to read than string interpolation, less error prone, and offers a bit more type-safety.